backend/matching-service/package.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "matching-service",
+    "version": "1.0.0",
+    "main": "index.js",
+    "scripts": {
+        "build": "tsc",
+        "test": "jest",
+        "lint": "eslint . --fix --no-error-on-unmatched-pattern && prettier --write --ignore-unknown .",
+        "start": "npm run build && node dist/server.js",
+        "dev": "nodemon src/server.ts"
+    },
+    "author": "",
+    "license": "ISC",
+    "description": "",
+    "dependencies": {
+        "axios": "^1.7.7",
+        "class-validator": "^0.14.1",
+        "cors": "^2.8.5",
+        "dotenv": "^16.4.5",
+        "express": "^4.21.0",
+        "express-async-errors": "^3.1.1",
+        "helmet": "^7.1.0",
+        "mongoose": "^8.6.3",
+        "passport": "^0.7.0",
+        "passport-jwt": "^4.0.1",
+        "winston": "^3.14.2"
+    },
+    "devDependencies": {
+        "@repo/eslint-config": "*",
+        "@repo/request-types": "*",
+        "@repo/typescript-config": "*",
+        "@testcontainers/mongodb": "^10.13.1",
+        "@types/cors": "^2.8.17",
+        "@types/dotenv": "^6.1.1",
+        "@types/express": "^4.17.21",
+        "@types/helmet": "^0.0.48",
+        "@types/jest": "^29.5.13",
+        "@types/jsonwebtoken": "^9.0.7",
+        "@types/node": "^22.5.5",
+        "@types/passport": "^1.0.16",
+        "@types/passport-jwt": "^4.0.1",
+        "@types/supertest": "^6.0.2",
+        "globals": "^15.9.0",
+        "jest": "^29.7.0",
+        "jsonwebtoken": "^9.0.2",
```

Comment:
this should not be a dev dependency

Reply:
which one? i copied frm question service

Reply:
jsonwebtoken

Reply:
oh sorry my bad, jsonwebtoken is used in question service for tests, not for core app logic, can ignore my prev message